libs/dal/src/repositories/layout/layout.repository.ts

```diff
@@ -76,6 +76,9 @@ export class LayoutRepository extends BaseRepository<EnforceEnvironmentQuery, La
           ...query,
         },
       },
+      {
+        $sort: { createdAt: -1 },
```

Comment:
Sort layout list by creation date

Reply:
I think we should do it by query param in the endpoint and here render the value passed. Let me do a task for that for doing it and don't apply this change in this PR. 🙂